src/Components/TodoItem.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <li class="flex">
+    <div class="justify-start item-id">
+      <slot name="id"></slot>
+    </div>
+    <div>
+      <slot name="description"></slot>
+    </div>
+  </li>
+</template>
+
+<script>
+export default {
+  name: "TodoItem",
+  props: {
+    item: {
```

Comment:
It seems like the props are unused as we are using slots <insert meaningful peer review comments here>

Reply:
Is this what you do in peer reviews?

Reply:
Yeah sure you can fix it in a new ticket :p